tests/unit/scripting.tcl

```diff
@@ -36,23 +36,16 @@ if {$is_eval == 1} {
 
 start_server {tags {"scripting"}} {
 
+    if {$is_eval eq 1} {
     test {Script - disallow write on OOM} {
-        r FUNCTION load lua f1 replace { redis.register_function{
-            function_name='f1',
-            callback=function() return redis.call('set', 'x', '1') end,
-            flags={'allow-oom'}
-        }}
```

Comment:
do we have another test for functions to check that they're not executed in oom condition if they didn't declare `allow-oom`?

Reply:
Yes on the `functions.tcl`: `FUNCTION - deny oom`.
This test was verifying that if a function does get executed on OOM then it can not do commands that mark with CMD_DENYOOM flag. But this test is now no longer relevant for functions.